target/linux/mediatek/dts/mt7986a-ruijie-rg-x60-pro.dts

```diff
@@ -105,6 +105,8 @@
 		reset-deassert-us = <100000>;
 		reset-gpios = <&pio 6 GPIO_ACTIVE_LOW>;
 		realtek,aldps-enable;
+		realtek,led-link-select = <0xa7 0x00 0x0>; /* led0 at 10/100/1000/2.5G/(2.5G lite)*/
```

Comment:
As I mentioned in your previous MR these two lines do not work. There is no such hack in OpenWRT for the realtek PHY driver:
~~~
abajkowski@abajkowski:~/workspace/openwrt$ grep -rni "realtek,led-link-select" .
abajkowski@abajkowski:~/workspace/openwrt$ grep -rni "realtek,led-act-select" .
abajkowski@abajkowski:~/workspace/openwrt$ 
~~~

Reply:
By the way, do you build an image with out-of-tree patches?